src/Layout/Footer.tsx

```diff
@@ -105,6 +106,9 @@ function Footer({}: Props): ReactElement {
               height={34}
             />
           </a>
+          <div>
+            <UNEPLogo/>
+          </div>
```

Comment:
Please put the logo in an anchor tag with target blank and href="https://www.unep.org/". Refer web-app code for more details.